core/blockchain.go

```diff
@@ -2366,3 +2379,7 @@ func (bc *BlockChain) SubscribeLogsEvent(ch chan<- []*types.Log) event.Subscript
 func (bc *BlockChain) SubscribeBlockProcessingEvent(ch chan<- bool) event.Subscription {
 	return bc.scope.Track(bc.blockProcFeed.Subscribe(ch))
 }
+
+func (bc *BlockChain) SupportsMultitenancy(context.Context) (*proto.PreAuthenticatedAuthenticationToken, bool) {
```

Comment:
`EthAPIBackend` implements the interface and it would extract the token from the context and return. 
The implementation is also extracted by `APIBackendHelper` to support `PrivateExtensionAPI` logic.